src/main/java/org/microg/nlp/backend/nominatim/BackendService.java

```diff
@@ -182,7 +210,7 @@ public void run() {
             synchronized (done) {
                 try {
                     Log.d(TAG, "Requesting " + url);
-                    HttpURLConnection connection = (HttpURLConnection) new URL(url)
+                    HttpsURLConnection connection = (HttpsURLConnection) new URL(url)
```

Comment:
I tend to use the least specific class that still suites my needs, in this case that would be `HttpURLConnection`. Just consider that a future Java version might decide that http 2.0 is not done using `HttpsURLConnection`, although it es encrypted. It's also very unlikely that we ever want to fiddle with tls parameters and finally using classes from the `javax` package should generally be done only if really needed.